dietpi/dietpi-software

```diff
@@ -652,6 +652,14 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/media/#jellyfin'
 		# - ARMv6: https://github.com/jellyfin/jellyfin/issues/5011
 		aSOFTWARE_AVAIL_G_HW_ARCH[$software_id,1]=0
+		#------------------
+		software_id=190
+
+		aSOFTWARE_NAME[$software_id]='Beets'
+		aSOFTWARE_DESC[$software_id]='music organizer and manager'
+		aSOFTWARE_CATX[$software_id]=2
+		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/media/#beets'
+		aSOFTWARE_DEPS[$software_id]='130'
```

Comment:
130 is actually Python pip, while here only the runtime is required. As we install the APT package which has all required Python modules as dependency, I think we can and should remove the dependency here.

It is btw possible to install it via `pip3 install beets`, which brings a newer version and then requires 130. Not sure whether it's worth it, as it needs to be compiled, hence requires the whole build-essentials stack as well.

Reply:
The Debian package is only 2 minor versions behind, so I think that we'll be fine using that.